.expeditor/scripts/omnibus_chef_build.ps1

```diff
@@ -28,7 +28,9 @@ $env:ARTIFACTORY_BASE_PATH="com/getchef"
 $env:ARTIFACTORY_ENDPOINT="https://artifactory-internal.ps.chef.co/artifactory"
 $env:ARTIFACTORY_USERNAME="buildkite"
 
-Write-Output "--- Install Chef Foundation"
+
+$CHEF_FOUNDATION_VERSION = $env:CHEF_FOUNDATION_VERSION
+Write-Output "--- Installing Chef Foundation $CHEF_FOUNDATION_VERSION"
 . { Invoke-WebRequest -useb https://omnitruck.chef.io/chef/install.ps1 } | Invoke-Expression; install -channel "current" -project "chef-foundation" -v $CHEF_FOUNDATION_VERSION
```

Comment:
how about using the `$env:CHEF_FOUNDATION_VERSION` notation here directly? i.e.
`. { Invoke-WebRequest -useb https://omnitruck.chef.io/chef/install.ps1 } | Invoke-Expression; install -channel "current" -project "chef-foundation" -v $env:CHEF_FOUNDATION_VERSION`

Reply:
@vkarve-chef
I tried the above fix. It throws the below error.

At C:\workdir\.expeditor\scripts\omnibus_chef_build.ps1:31 char:45
--
  | + Write-Output "--- Install Chef Foundation : $env:${CHEF_FOUNDATION_VE ...
  | +                                             ~~~~~
  | Variable reference is not valid. ':' was not followed by a valid variable name
  | character. Consider using ${} to delimit the name.
  | At C:\workdir\.expeditor\scripts\omnibus_chef_build.ps1:32 char:152
  | + ... nstall -channel "current" -project "chef-foundation" -v $env:$CHEF_FO ...
  | +                                                             ~~~~~
  | Variable reference is not valid. ':' was not followed by a valid variable name
  | character. Consider using ${} to delimit the name.
  | + CategoryInfo          : ParserError: (:) [], ParseException
  | + FullyQualifiedErrorId : InvalidVariableReferenceWithDrive
  ========
  
  Trying with adding  ${}

Reply:
. { Invoke-WebRequest -useb https://omnitruck.chef.io/chef/install.ps1 } | Invoke-Expression; install -channel "current" -project "chef-foundation" -**v ${env:CHEF_FOUNDATION_VERSION}** - went well with this changes

Reply:
the change looks good to me and simplifies binding the variable.